inappmessaging/src/test/resources/test_response.json

```diff
@@ -5,6 +5,9 @@
       "campaignData": {
         "campaignId": "1234567890",
         "isTest": false,
+        "hasNoEndDate": true,
```

Comment:
add test case where there are no optional params

Reply:
@vincentdelmundo But those params are not optional. (Regardless what backend docs say)

Reply:
yes, the parameters have default values set in the data class (already covered in other UT)